mountpoint-s3/scripts/fs_bench.sh

```diff
@@ -36,6 +36,10 @@ if [[ -n "${S3_ENDPOINT_URL}" ]]; then
   optional_args+="--endpoint-url=${S3_ENDPOINT_URL}"
 fi
 
+if [[ -n "${S3_DEBUG}" ]]; then
+  optional_args+= " --debug"
+fi
```

Comment:
nit (and in the other files)

```suggestion
if [[ -n "${S3_DEBUG}" ]]; then
  optional_args+=" --debug"
fi
```

Reply:
thanks, will fix.